### ISSUE TITLE
"Add Project Folder…" menu item does nothing if no window is open 

### ISSUE THREAD

**Issue by redfood** [atom/atom#6995]
I am using Atom Version 0.204.0 (0.204.0) on OS X 10.10.3.

If there isn't a window open but you select "Add Project Folder…" from the "File" menu nothing happens.

The expected behavior would be for a file selection dialog box to appear and then to have a new window open with the selected folder added as a project.

(Alternatively, the menu item could be greyed out I suppose but having it operational would match the behavior of the "Open…" menu item.

**Comment by samuela** [comment]
Yeah, why doesn't it just open a new window with the selected folder?

**Comment by dr-impossible** [comment]
I agree, the current behaviour is pretty annoying

**Comment by redfood** [comment]
This is still a bug 2 years later.  Bumping it so that it isn't closed because it "stale."

**Comment by redfood** [comment]
I can still reproduce the issue in the latest version of Atom.

I am using now using Atom 1.28.1 x64
On MacOS Sierra version 10.12.6 (16G1314)

The steps I've followed to reproduce the issue are the same as those outlined in the original bug report:
1) Close all open Atom windows.
2) Chose File > "Add Project Folder…"

Alternatively:
1) Close all open Atom windows.
2) Press shift-cmd-O

3 years, 1 month, 13 days since the original bug report no response (except from the stale-bot).

**Comment by redfood** [comment]
Also - I'm not sure why this was labled an "enhancement".  Its clearly a **bug**.  Is the mislabling the reason why it is being ignored?

**Comment by rsese** [comment]
It looks like https://github.com/atom/atom/issues/12508 got a lot of traction and reports about the various `File` menu options not working with no windows open on macOS got redirected there.  It also looks like there's a [pull request](https://github.com/atom/atom/pull/17529) opened to resolve https://github.com/atom/atom/issues/12508 but I don't think it addresses `File > Add Project Folder` - if `File > Open` worked though, would you feel like `File > Add Project Folder` would still be necessary to work since `File > Open` would behave as you described?

> The expected behavior would be for a file selection dialog box to appear and then to have a new window open with the selected folder added as a project.

**Comment by redfood** [comment]
rsese - thanks for the response and changing the label!

Would "File > Add Project Folder" "still be necessary to work since File > Open would behave as you described?"    Necessary?  Atom is quite usable as is. This obviously isn't a show-stopping bug. I just reported it because the current UI behavior is unexpected on the Mac.  Fixing the Open menu item would still leave the "Add Project Folder" behaving incorrectly (at least on the Mac).  

Thanks for giving my pet bug some attention!

**Comment by rsese** [comment]
> Fixing the Open menu item would still leave the "Add Project Folder" behaving incorrectly (at least on the Mac).

Yeah that's true - I guess I'm wondering if `Add Project Folder` can just be disabled since `Open` would be available and work the same if https://github.com/atom/atom/pull/17529 is merged.  Though there's no timeline for that pull request to get reviewed, I'll mention this to the team and see what they think.

**Comment by redfood** [comment]
Yes - I think disabling the menu item (graying it out) when no windows are open would be a proper fix.
